docs/decisions/0007-release-process.md

```diff
@@ -0,0 +1,334 @@
+
+   git commit -s -S -m "update submodules for v0.11.0-rc1"
+   git submodule update --init --remote --sync
+   ```
+
+4. Tag the `s3gw` repository with the appropriate release candidate tag.
+
+   ```shell
+   git tag --annotate --sign -m "Release Candidate 1 for v0.11.0" s3gw-v0.11.0-rc1
+   ```
+
+5. Push the release branch and tag. This will trigger the release pipeline,
+   creating the various release artifacts and a draft release.
+
+   ```shell
+   git push upstream s3gw-v0.11
+   ```
+
+6. Once the containers have been created and pushed to Quay, it's time to start
+   validating the release candidate. Please refer to the
+   [Testing Section](#testing) before continuing.
+
+7. If any patches needed to be backported at some point since the last release
+   candidate, please go back to `step 2.` and increase the release candidate
+   version by `1` (i.e., `-rc2`, `-rc3`, etc.). Even if a particular sub-project
+   repository has not been changed, it is still crucial to tag it with the new
+   release candidate version, for consistency across repositories.
+
+8. Assuming everything goes well, we can now go through step `2.` but, instead
+   of tagging for a release candidate version, we will be tagging for the
+   release version.
+
+   ```shell
+   git tag --annotate --sign -m "Release v0.11.0" s3gw-v0.11.0
+   git push upstream s3gw-v0.11
+   ```
+
+9. At this point we will need to update the Helm Chart to reflect the release
+   version. This becomes bit tricky, because we want the change to be reflected
+   in both the `main` branch and the `s3gw-vX.Y` branch on the `s3gw-charts`
+   repository. To do this, we will apply a patch to the `main` branch, and then
+   backport the change to the release branch.
+
+   First, for `v0.11.0`, the chart version needs to be updated with the specific
+   version, at `charts/s3gw/Chart.yaml`. This change should then be committed,
+   and a Pull Request of this change opened against `main`.
+
+   Once the Pull Request has been merged, note down the new commit's `SHA`;
+   running `git log` should give you its value. We can now change to the
+   `s3gw-v0.11` branch, and `cherry-pick` the commit, and finally tag the branch
+   with the release version.
+
+   ```shell
+   git cherry-pick -x -s -S <SHA>
+   git tag --annotate --sign -m "Release v0.11.0" s3gw-v0.11.0
+   git push upstream s3gw-v0.11
+   ```
+
+10. With all sub-project repositories ready to be released, it's time to prepare
+    the `s3gw` repository for a release. Much like what we did for the
+    `s3gw-charts` repository, we will have to apply a patch on the `main` branch
+    first, and then backport it to the release branch: this time to keep track
+    of the `CHANGELOG`.
+
+    First step, we need to go to the [Current CHANGELOG][8] page on the
+    repository's Wiki, and copy the contents for the release version being
+    handled to a file at `docs/release-notes/s3gw-vX.Y.Z.md`. Keep in mind the
```

Comment:
Also update the symlink `docs/release-notes/s3gw-latest.md` to point to the latest release notes.